extension/src/test/suite/helper.ts

```diff
@@ -10,3 +11,12 @@ export function princ(str : any){
     var value = str[name];
 	console.log(`-----the ${name} is ${value.toString()}-----`);
 }
+//This method is to read a file and return the vscode.TextDocument object 
+export async function readFile2TextDocument(filepath: string) : Promise<vscode.TextDocument | undefined> {
+	let doc : vscode.TextDocument;
+	try {
+        return doc = await vscode.workspace.openTextDocument(filepath);
+      } catch (error) {
+        return undefined;
```

Comment:
how about console.log(error) before return?

Reply:
Yes, I will add this.